docs_src/index.md

```diff
@@ -1,34 +1,25 @@
-# Vision Self Checkout
+# Automated Self Checkout
 
-## Introduction
-This guide helps you build and run the Vision Self Checkout solution.
+## Overview
 
-Upon completing the steps in this guide, you will be ready to run and benchmark pipelines on different hardware setups.
+The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy a self-checkout use case using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured automated self checkout pipeline that is optimized for Intel® hardware.  The reference solution also includes a set of benchmarking tools to evaluate the workload on different hardware platforms. Together the reference solution will help evaluate your required hardware to minimize the cost per workload.
```

Comment:
are you currently providing components to build/deploy for the self-checkout use case? or is it mostly benchmarking?

Reply:
It is a mix of both but most of it is tied to the benchmarking scripts